dev-tools/cherrypick_pr

```diff
@@ -135,6 +135,9 @@ def main():
         session.post(
             base + "/issues/{}/labels".format(new_pr["number"]), json=["backport", "review"])
 
+        # remove needs backport label from the original PR
+        session.delete(base + "/issues/{}/labels/needs_backport".format(args.pr_number))
```

Comment:
I agree it's not ideal. In case we need to backport a PR to multiple branches, then it would be nice to enhance the script to be able to pass a list of branches instead of one, and then remove the `needs_backport`, but this is for another PR.